addons/mail/static/src/models/rtc_data_channel/rtc_data_channel.js

```diff
@@ -0,0 +1,24 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr } from '@mail/model/model_field';
+
+registerModel({
+    name: 'RtcDataChannel',
+    identifyingFields: ['token'],
```

Comment:
same comment as in other PRs, should probably have session record relation as identifying field

Reply:
I would still much prefer if we cleaned this up before merging, to use proper identifying field (I think it should be a one2one with rtc session, which is the holder of the token).

Reply:
I can't just define a `one2one` with `RtcSession` with current code, because it makes use of many `dataChannels` while there's currently just a `one` with `Rtc/currentRtcSession`.

Are you sure `RtcDataChannel/token` is conceptually the same as `RtcSession/id`?
- If yes, I could do some changes to go in direction of what you say, like defining `Rtc/rtcSessions` that is a `many` of all known `RtcSession`s.
- If not sure, then it's much safer to keep mode isolated for the time being, and reconcile at a later date when the code will make far more sense when they are conceptually the same.

Reply:
Yes, `token`/`peerToken` is `RtcSession/id`, It a remnant from when we used channel members instead of "temporary" rtc sessions, I should have refactored it.

Reply:
> Yes, `token`/`peerToken` is `RtcSession/id`, It a remnant from when we used channel members instead of "temporary" rtc sessions, I should have refactored it.

Cool, so will refactor so that all `token`/`peerToken` dicts are instead record that are identified by the `RtcSession` (since `RtcSession/id` is identifying field of `RtcSession`).